Assets/MixedRealityToolkit.Tests/PlayModeTests/PointerTests.cs

```diff
@@ -101,15 +101,96 @@ public IEnumerator TestLinePointers()
         }
 
 
+        /// <summary>
+        /// Test pointers are correctly enabled when interacting with colliders that are visible, but whose
+        /// bounds are outside the camera FOV.
+        /// </summary>
+        [UnityTest]
+        public IEnumerator TestPointerFieldOfViewLargeCollider()
+        {
+            var rightHand = new TestHand(Handedness.Right);
+            yield return PlayModeTestUtilities.WaitForInputSystemUpdate();
+
+            var cube = GameObject.CreatePrimitive(PrimitiveType.Cube);
+            cube.AddComponent<NearInteractionGrabbable>();
+            cube.AddComponent<NearInteractionTouchableVolume>();
+            yield return rightHand.Show(Vector3.zero);
+            yield return PlayModeTestUtilities.WaitForInputSystemUpdate();
+
+            var spherePointer = PointerUtils.GetPointer<SpherePointer>(Handedness.Right);
+            var pokePointer = PointerUtils.GetPointer<PokePointer>(Handedness.Right);
+
+            yield return TestPointerFieldOfViewLargeColliderHelper(spherePointer, cube, rightHand);
+            yield return TestPointerFieldOfViewLargeColliderHelper(pokePointer, cube, rightHand);
+
+            rightHand.Hide();
+            GameObject.Destroy(cube);
+        }
+
+
+        private IEnumerator TestPointerFieldOfViewLargeColliderHelper(IMixedRealityPointer myPointer, GameObject cube, TestHand testHand)
```

Comment:
nit: very minor but maybe TestPointerFOVLargeColliderHelper to simplify a bit

Reply:
Also nit: can this be placed at bottom of class as private?

Reply:
Done